public/configs/config-cleveland.hjson

```diff
@@ -182,38 +186,38 @@
                 // location: 'sidebar' | 'aboveStimulus' | 'belowStimulus'
                 location: 'sidebar'
             }
-//text example
-                {
-                    id: "cm-response2"
-                    type: "numerical"
-                    prompt: "Your answer!!"
-                    desc: "Enter your answer here, range from 0 - 100"
-                    required: true
-                    // add max and min for the allowed range
-                    max: 100
-                    min: 0
-                    location: aboveStimulus
-                }
-//likert example
-                {
-                    id: "cm-response3"
-                    type: "likert"
-                    prompt: "Your answer!!"
-                    desc: "Enter your likert answer here"
-                    required: true
-                    //specify the number of likert options.
-                    preset: "5"
-                    location: belowStimulus
-                }
-//text example
-                {
-                    id: "cm-response4"
-                    type: "shortText"
-                    prompt: "Your answer!!"
-                    desc: "Enter your short text answer here"
-                    required: true
-                    location: aboveStimulus
-                }
+////text example
+//                {
+//                    id: "cm-response2"
+//                    type: "numerical"
+//                    prompt: "Your answer!!"
+//                    desc: "Enter your answer here, range from 0 - 100"
+//                    required: true
+//                    // add max and min for the allowed range
+//                    max: 100
+//                    min: 0
+//                    location: aboveStimulus
+//                }
+////likert example
+//                {
+//                    id: "cm-response3"
+//                    type: "likert"
+//                    prompt: "Your answer!!"
+//                    desc: "Enter your likert answer here"
+//                    required: true
+//                    //specify the number of likert options.
+//                    preset: "5"
+//                    location: belowStimulus
+//                }
+////text example
+//                {
+//                    id: "cm-response4"
+//                    type: "shortText"
+//                    prompt: "Your answer!!"
+//                    desc: "Enter your short text answer here"
+//                    required: true
+//                    location: aboveStimulus
+//                }

```

Comment:
Let's remove this before merge

Reply:
Done. Change questions to real survey questions.